milvus/HttpClient.ts

```diff
@@ -12,45 +12,9 @@ export class HttpBaseClient {
   // The client configuration.
   public config: HttpClientConfig;
 
-  // axios
-  public client: AxiosInstance;
-
   constructor(config: HttpClientConfig) {
```

Comment:
@shanghaikid I would suggest allowing optional users to pass in a fetch library into this client.  A lot of serverless environments have their own fetch pecularities, and it might be better to offer users flexibility.  In that case having the optional ability to use another fetch so that it works properly with the serverless environment might work better.

This way the test issues encountered (node-fetch v2 vs node-fetch v3 in jest) need not impact runtime.